File: javascript/1. Two Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {

  for ( const [idx, val] of nums.entries()) {
    const idx2 = nums.indexOf(target - val, idx + 1);

    if (idx2 != -1) {
      return [idx, idx2];
    }
  }
};

File: python/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for idx, val in enumerate(nums):
            try:
                idx2 = nums.index(target-val, idx+1)
                
                return [idx, idx2]
            except Exception:
                pass